libqtile/command/base.py

```diff
@@ -40,6 +42,50 @@
     ItemT = Optional[tuple[bool, list[str | int]]]
 
 
+def expose_command(name: Callable | str | list[str] | None = None) -> Callable:
+    """
+    Decorator to expose commands to IPC.
```

Comment:
Maybe to clarify it's not just IPC but also the lazy interface?
```suggestion
    Decorator to expose methods to the command interface.
```